### ISSUE TITLE
Component: progressbar

### ISSUE THREAD

**Issue by Philipaplatt** [primefaces/primeng#12212]
### Describe the bug

As reported by Axe DevTools for 508 compliance

"ARIA progressbar must have an accessible name"   
https://dequeuniversity.com/rules/axe/4.4/aria-progressbar-name?application=AxeChrome

Element Source

`<div role="progressbar" aria-valuemin="0" aria-valuemax="100" ng-reflect-ng-class="[object Object]" aria-valuenow="4" class="p-progressbar p-component p-progressbar-determinate">`

This is solved by adding an aria-label attribute to the template 

### Environment

We noticed this on a work project. Since we cannot pull (blocked by firewall), I replicated the issue on my personal machine (desktop). Axe DevTools was used to identify the issue. Using visual studio code, just pulled down the source and tested with adding the aria label

### Reproducer

Just run Axe DevTools scan(free version, scan entire page) over the progress bar page. It is the top issue reported.

### Angular version

14.0.7

### PrimeNG version

14.2.2-SNAPSHOT

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

16.13.2

### Steps to reproduce the behavior

1. Install Axe DevTools free chrome extension:  https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US
2. Navigate to   http://localhost:4200/progressbar
3. Go to browser dev tools, click the double chevrons in the upper right (>>), choose 'axe DevTools'
4. Click the button "scan all of my page"
5. Find the issue reported "ARIA progressbar must have an accessible name", should be the first one

### Expected behavior

Axe DevTools should not report a serious accessibility problem, this is solved by adding the aria-label attribute to the progress bar

**Comment by mehmetcetin01140** [comment]
Hi, 

We apologize for the delayed response. Our team began implementation for accessibility with version 16.5x and successfully completed the accessibility implementation with version 16.9.1 after intensive work and meticulous study. The mentioned issue might have been resolved between these versions. If the problem persists in PrimeNG v17, please feel free to open a new issue without hesitation. Thanks for reporting the issue!
